cross-chain/solana/package.json

```diff
@@ -1,7 +1,15 @@
 {
+    "name": "@keep-network/tbtc-v2-solana",
+    "version": "1.0.0-dev",
+    "description": "tBTC v2 on Solana",
+    "license": "GPL-3.0-only",
     "scripts": {
+        "build": "anchor build",
+        "format": "npm run lint",
+        "format:fix": "npm run lint:fix",
         "lint:fix": "prettier */*.js \"*/**/*{.js,.ts}\" -w",
-        "lint": "prettier */*.js \"*/**/*{.js,.ts}\" --check"
+        "lint": "prettier */*.js \"*/**/*{.js,.ts}\" --check",
+        "test": "anchor test"
```

Comment:
I would get rid of this, too, since you have the Makefile specifying the `test` command.

Reply:
Sure, done in [feb345b](https://github.com/keep-network/tbtc-v2/pull/688/commits/feb345b5cf7bd4affe636bd3fe691a7f066511f6)